boring/src/ssl/mod.rs

```diff
@@ -712,25 +712,21 @@ impl SslContextBuilder {
     }
 
     /// Sets raw public key certificate in DER format.
-    pub fn set_rpk_certificate(&mut self, cert: &[u8]) -> Result<(), ErrorStack> {
+    pub fn set_raw_public_key_mode(&mut self) {
         unsafe {
-            cvt(ffi::SSL_CTX_set_server_raw_public_key_certificate(
-                self.as_ptr(),
-                cert.as_ptr(),
-                cert.len() as u32,
-            ))
-            .map(|_| ())
+            ffi::SSL_CTX_set_raw_public_key_mode(self.as_ptr());
         }
     }
 
     /// Sets RPK null chain private key.
-    pub fn set_null_chain_private_key<T>(&mut self, key: &PKeyRef<T>) -> Result<(), ErrorStack>
+    pub fn set_raw_public_key_and_key<T>(&mut self, key: &PKeyRef<T>) -> Result<(), ErrorStack>
```

Comment:
`set_raw_public_key_and_key` -  the name suggest that this now also need to set a public key cert along with private one, no?

Reply:
the public key is automatically generated from the private key and stored on SSL_CTX. https://boringssl-review.googlesource.com/c/boringssl/+/45444/18 see docs on the function header